src/app/asset-grid/thumbnail/thumbnail.component.pug

```diff
@@ -1,27 +1,16 @@
-.card-block.card-text(tabindex="3", [attr.aria-label]="thumbnail.tombstone && thumbnail.tombstone.length > 0 ? thumbnail.tombstone.join(', ') : thumbnail.name + ', '  + thumbnail.agent + ', ' + thumbnail.date", [attr.id]="('item-' + itemIndex)", [ngClass]="{ 'reorder--asset' : arrowReorderMode }")
```

Comment:
@codiak Can you please review this line? I removed the check for `thumbnail.tombstone` and this whole ternary for the aria label. The `thumbnail.tombstone ` will throw property does not exist error on prod build.